classes/submission/genre/Genre.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace PKP\submission\genre;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Casts\Attribute;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\Facades\DB;
+use PKP\submissionFile\SubmissionFile;
+
+
+
+class Genre extends Model
+{
+    const GENRE_CATEGORY_DOCUMENT = 1;
+    const GENRE_CATEGORY_ARTWORK = 2;
+    const GENRE_CATEGORY_SUPPLEMENTARY = 3;
+    protected $table = 'genres';
+    protected $primaryKey = 'genre_id';
+    public $timestamps = false;
+
+    protected $fillable = [
+        'context_id', 'seq', 'enabled', 'category', 'dependent', 
+        'supplementary', 'required', 'entry_key'
+    ];
+
+    // Relationships
+    public function submissionFiles(): HasMany
+    {
+        return $this->hasMany(SubmissionFile::class, 'genre_id');
```

Comment:
Are you sure this should be `genre_id`?

Reply:
I  used this  relationship definition because in the database schema, `genre_id` is used as the foreign key in `submission_files`.

Reply:
Can you point me to the code you were looking at? I don't think it's correct.

Reply:
:+1: I misread this, thanks!